src/data/faq_de.json

```diff
@@ -1996,5 +1996,141 @@
                 ]
             }
         ]
-    }
+    },
+    "glossary": {
+        "glossaryHeadline": "Glossariung",

```

Comment:
Or maybe just "Glossar" ?

Reply:
Yes, please! 😄

Reply:
Sorry the content is still in redaction, this is just dummy content ;)

Reply:
Thanks for changing to "Draft" status. That makes it clearer now.